cloudhub-2/modules/ROOT/pages/ps-assign-permissions.adoc

```diff
@@ -9,7 +9,7 @@ To assign permissions, you need access to Access Management and the Organization
 * To view a private space, you need the CloudHub Network Viewer permission.
 * To deploy applications from Exchange, you need the Exchange Contributor and Exchange Viewer permissions.
 * To deploy applications in Runtime Manager, you need the Create Applications permission.
-* To view all applications on the *Applications* page in Runtime Manager, you need the Read Applications and Read Runtime Fabric permissions.
+* To view all applications on the *Applications* page in Runtime Manager, you need the Read Applications and CloudHub Network Viewer permission.
```

Comment:
To view all applications on CH2 target ?

Reply:
to be inline with other wording we can keep the same message instead of CH2 target